apps/passport-server/src/services/telemetryService.ts

```diff
@@ -45,7 +47,7 @@ export async function startTelemetry(
       writeMarker(
         context.gitCommitHash,
```

Comment:
i've been meaning to change this to the first 20ish characters of the git commit message, which is loaded elsewhere on the application startup path, could you do that for me here?